test/vinyl-luatest/gh_7240_abort_yielding_dml_test.lua

```diff
@@ -0,0 +1,42 @@
+local misc = require('test.luatest_helpers.misc')
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new{alias = 'master'}
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+g.test_abort_yielding_dml = function(cg)
```

Comment:
Please add a brief comment explaining what is tested.

Reply:
Done.